lib/pages/chat_draft/draft_chat_view.dart

```diff
@@ -38,25 +38,22 @@ class DraftChatView extends StatelessWidget {
         toolbarHeight: DraftChatViewStyle.toolbarHeight(context),
         surfaceTintColor: Colors.transparent,
         automaticallyImplyLeading: false,
-        title: Padding(
-          padding: DraftChatViewStyle.paddingTopScreen(context),
-          child: Row(
-            children: [
-              TwakeIconButton(
-                tooltip: L10n.of(context)!.back,
-                icon: Icons.arrow_back,
-                onPressed: () => context.pop(),
-                paddingAll: 8.0,
-                margin: const EdgeInsets.symmetric(vertical: 12.0),
+        title: Row(
+          children: [
+            TwakeIconButton(
+              tooltip: L10n.of(context)!.back,
+              icon: Icons.arrow_back,
+              onPressed: () => context.pop(),
+              paddingAll: 8.0,
+              margin: const EdgeInsets.symmetric(vertical: 12.0),
+            ),
+            Expanded(
+              child: _EmptyChatTitle(
```

Comment:
don't use controller inside StatelessWidget

Reply:
Written by `sherlockvn` 😁

Reply:
- Removed `controller` inside stl, I move it to `argument` pls check it